src/terms/xc.jl

```diff
@@ -182,17 +221,21 @@ end
 function max_required_derivative(functional)
     functional.family == :lda && return 0
     functional.family == :gga && return 1
+    if functional.family == :mgga
+        return needs_laplacian(functional) ? 2 : 1
+    end
     error("Functional family $(functional.family) not known.")
 end
 
 
 # stores the input to libxc in a format it likes
 struct LibxcDensity
-    basis
+    basis::PlaneWaveBasis
     max_derivative::Int
     ρ_real    # density ρ[iσ, ix, iy, iz]
     ∇ρ_real   # for GGA, density gradient ∇ρ[iσ, ix, iy, iz, iα]
     σ_real    # for GGA, contracted density gradient σ[iσ, ix, iy, iz]
+    Δρ_real   # for (some) mGGA, Laplacian of the density Δρ[iσ, ix, iy, iz]
 end
```

Comment:
why isn't tau in this list?

Reply:
Because it's not a density derivative, but generated from the orbitals. Physically that sort of lives in a different compartment.

Reply:
meh, code organization trumps physical relevance here I would say, but as you wish. Another option would be to rename the struct to LibxcInput

Reply:
Yeah, ok, that's a good point. I'll think about it.